source/game.screen.supermarket.production.bmx

```diff
@@ -1666,9 +1668,34 @@ Type TGUISelectCastWindow Extends TGUIProductionModalWindow
 		castSelectList.filteredGenderID = filterToGenderID
 		'initial sorting
 		castSelectList.entries.sort(True, TGUICastSelectList.SortCastByName)
+
+		'instead of yes/no, this could return the age...
+		'TODO this filter does not prevent moving an underage actor to a director slot
+		Function filterAge:Int()
+			Local handler:TScreenHandler_SupermarketProduction = TScreenHandler_SupermarketProduction.GetInstance()
+			Local script:TScriptBase = handler.currentProductionConcept.script
+			Local jobSlotList:TGUICastSlotList = handler.castSlotList
+			Local jobId:Int = jobSlotList.selectCastWindow.selectJobID
+
+			If jobId = TVTPersonJob.DIRECTOR Then Return True
+			If jobId = TVTPersonJob.HOST Then Return True
```

Comment:
Es gibt denke ich Kindersendungen, in denen der "Moderator" oder "Reporter" minderjaehrig ist.

Denke da beispielsweise an die "ersten Ausgaben" von "Anna und die wilden Tiere".